haralyzer/assets.py

```diff
@@ -632,3 +632,84 @@ def video_load_time(self):
     @cached_property
     def html_load_time(self):
         return self._get_asset_load('html')
+
+
+class HarEntry(object):
+    """
+        An object that represent one entry in a HAR Page
+    """
+    def __init__(self, entry):
+        self.raw_entry = entry
+
+    def get_header_value(self, name):
+        """
+        Returns the header value of the header defined in ``name``
+
+        :param name: ``str`` name of the header to get the value of
+        """
+        for x in self.headers:
+            if x["name"].lower() == name.lower():
+                return x["value"]
+
+    @cached_property
+    def headers(self):
```

Comment:
It feels weird to me to return ALL of the headers (both response and request), what is the use case here? I would assume that most of the time you would be interested in one or the other specifically. Also, what happens if there are key collisions here?

Reply:
This is left over, before I split up request and response in to seperate classes. Will be removed.